src/Core/src/Platform/Windows/MauiWinUIWindow.cs

```diff
@@ -96,5 +112,35 @@ IntPtr NewWindowProc(IntPtr hWnd, uint msg, IntPtr wParam, IntPtr lParam)
 		}
 
 		#endregion
+
+		/// <summary>
+		/// Default the Window Icon to the icon stored in the .exe, if any.
+		/// 
+		/// The Icon can be overriden by callers by calling SetIcon themselves.
+		/// </summary>
+		void SetIcon()
```

Comment:
Should we be loading and caching this image for multiple windows? Then we could potentially leave out the cleanup...

Reply:
Do we support multiple windows?

Reply:
yap